src/workloads/execution/CreateIndexSharded.yml

```diff
@@ -44,23 +23,20 @@ Actors:
   - *Nop
   - *Nop
 
-- Name: CreateShardKeyIndex
```

Comment:
Since the collection is empty when we shard the collection, we don't need a phase for creating the index.